app/src/main/res/values/styles.xml

```diff
@@ -8,4 +8,13 @@
         <item name="colorAccent">@color/colorAccent</item>
     </style>
 
+    <style name="AppTheme.NoActionBar">
```

Comment:
I don't think you need this right now, why are they part of the PR?

Reply:
It is for the action bar

Reply:
ah, okay...🤔